bundle/jsky.app.ot/src/main/java/jsky/app/ot/util/Resources.java

```diff
@@ -96,14 +97,16 @@ public static ImageIcon getIcon(String iconFileName) {
     }
 
     public static void setOTFrameIcon(java.awt.Frame frame) {
-        ImageIcon icon = Resources.getIcon("ot.png");
-        if (icon != null) {
-            Image image = icon.getImage();
-            frame.setIconImage(image);
+        if (Platform.get() != Platform.osx) {
+            // Set the frame icon. This fails in OSX but it is useful on Windows and Linux
```

Comment:
Can you elaborate on "fail" a little bit? The next person to look at this will be curious.